messaging/propagate.go

```diff
@@ -73,7 +74,7 @@ type propagationContext interface {
 func NewPropagationFunc(c propagationContext, name string, f PropagationStore) (PropagationFunc, error) {
 	pid, err := c.ProtocolRegister(name, func(n *onet.TreeNodeInstance) (onet.ProtocolInstance, error) {
 		p := &Propagate{
-			sd:               &PropagateSendData{[]byte{}, 1000},
+			sd:               &PropagateSendData{[]byte{}, 100000},
```

Comment:
Can't this be a global variable ? Looks like a good fit (for once). Otherwise one can add an arguement to take the timeout but i don't think that's necessary. What do you think ?